src/benchmark/Measurement.js

```diff
@@ -0,0 +1,34 @@
+class Measurement {
+
+    constructor(warmupRounds, measureRounds) {
+        this.warmupRounds = warmupRounds;
+        this.measureRounds = measureRounds;
+    }
+
+    executeMeasurement(jsAlgorithm) {
+        this.executeAlgorithm(jsAlgorithm, this.warmupRounds);
+        let jsTime = this.measureTime(jsAlgorithm, this.measureRounds);
+
+        // cross compile here
+        let wasmAlgorithm = jsAlgorithm; // change to cross compiled function
+        this.executeAlgorithm(wasmAlgorithm, this.warmupRounds);
+        let wasmTime = this.measureTime(wasmAlgorithm, this.measureRounds);
+
+        return {jsTime, wasmTime};
+    }
+
+    measureTime(algorithm) {
```

Comment:
`measureTime` only returns the time of all executions.
It would be better to return an array with an element for each round.